python/astshim/test.py

```diff
@@ -90,7 +90,9 @@ def checkRoundTrip(self, amap, poslist, rtol=1e-05, atol=1e-08):
         # test vector versions of forward and inverse
         posvec = list(poslist.flat)
         to_posvec = amap.tranForward(posvec)
-        assert_allclose(to_posvec, list(to_poslist.flat), rtol=rtol, atol=atol)
+        # cast to_poslist to np.array because if poslist has 1 axis then
+        # a list is returned, which has no `flat` attribute
```

Comment:
Good catch. The problem was caused by the input data being a list instead of a list of lists. That works but calls the vector version, which returns a list, which doesn't have the `flat` attribute. I have implemented a more elegant fix: check the size of poslist and make it 2d if necessary.